test/data/openshift/get_user.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
What is this used for?

Reply:
what part of testing the `whoami` function (older version of the parent issue), but I forgot to remove it. Let me do it now.